### PR TITLE
Add fee breakup in order notes

### LINKED ISSUE #2522: Display fee breakdown in order notes.
Splitted from: https://github.com/Automattic/woocommerce-payments/issues/2233
All transactions are subject to a base fee (determined by the merchant's country) plus a number of additional fees (like international card processing, FX fees etc).

We currently only show the total fee. We want to break this number down to make it clearer how the final rate was calculated.

**Order Notes**
image

### PR DESCRIPTION
**Pull request by naman03malhotra**
Fixes #2522 

#### Testing instructions

- Need to be tested with server `fx-fees-breakdown`

*

-------------------

- [ ] Added changelog entry (or does not apply)
- [ ] Covered with tests (or have a good reason not to test in description ☝️)
- [ ] Tested on mobile (or does not apply)

### CODE REVIEW

**Comment by naman03malhotra** [comment]
More context on progress and handover here: https://a8c.slack.com/archives/C01BPL3ALGP/p1627373063249700?thread_ts=1627365438.244100&cid=C01BPL3ALGP

**Comment by naman03malhotra** [comment]
Update:
Discussed this issue during team chat: https://gammap2.wordpress.com/2021/07/28/gamma-chat-wed-july-28th-1000-utc/

tldr; Probably worth solving this with https://github.com/Automattic/woocommerce-payments/issues/1054. Final call will be taken by @RadoslavGeorgiev

**Comment by htdat** [comment]
Closing, the new PR to fix the same issue #4212
